terraform/main.tf

```diff
@@ -50,3 +49,15 @@ module "psn" {
   subnet_ids         = ["${data.terraform_remote_state.gsp_cluster.subnet_ids[0]}", "${data.terraform_remote_state.gsp_cluster.subnet_ids[1]}"]
   security_group_ids = ["${data.terraform_remote_state.gsp_cluster.worker_security_group_id}"]
 }
+
+resource "aws_route53_record" "dcs_dev" {
+  zone_id = "${data.terraform_remote_state.gsp_cluster.r53_zone_id}"
+  name    = "dcs-dev.${data.terraform_remote_state.gsp_cluster.cluster_domain}"
+  type    = "A"
+
+  alias {
+    name                   = "nlb.${data.terraform_remote_state.gsp_cluster.cluster_domain"}"
```

Comment:
this has to be the output from the NLB itself; which would need to be exported from the upstream terraform project. urgh. it's probably not worth all the effort, let's make it a CNAME after all.